modules/node-tuning-operator-supported-tuned-daemon-plug-ins.adoc

```diff
@@ -34,9 +34,9 @@ that is not supported. The following TuneD plugins are currently not supported:
 * systemd
 
 
-[WARNING]
+[NOTE]
 ====
-The TuneD bootloader plugin is currently supported on {op-system-first} 8.x worker nodes. For {op-system-base-full} 7.x worker nodes, the TuneD bootloader plugin is currently not supported.
+The TuneD bootloader plugin is only supported on {op-system-first} worker nodes.
```

Comment:
```suggestion
The TuneD boot loader plugin only supports {op-system-first} worker nodes.
```

Reply:
will it be supported on RHEL 8.x worker nodes ? or is it only for COREOS